includes/footer.php

```diff
@@ -170,6 +152,15 @@
         ?>
 
         <script>
+            if(/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)){
```

Comment:
Honestly, I think the old implementation was _fine_... like Alex said, Bootstrap classes would be better, but plain CSS would be better than more scripts.